tests/gtest_like_c.h

```diff
@@ -0,0 +1,50 @@
+#ifndef GTEST_LIKE_C_H
+#define GTEST_LIKE_C_H
+
+#include <setjmp.h>
+#include <stdio.h>
+#include <string.h>
+
+#define TEST_NAME(GROUP, NAME) \
+    GROUP_##NAME
+
+#define TEST(GROUP, NAME) \
+    void TEST_NAME(GROUP,NAME)()
+
+extern jmp_buf gtest_jmp;
+
+enum {TEST_PASSED, TEST_FAILED, TEST_SKIPPED};
+
+#define ASSERT_EQ(VALUE1, VALUE2) \
```

Comment:
We'd use floating point numbers comparator as well (https://github.com/google/googletest/blob/main/docs/reference/assertions.md#expect_near-expect_near).

Reply:
Once it is needed it will be implemented